11.3.методы_списков_часть1/11.3.10.py

```diff
@@ -0,0 +1,6 @@
+numarr = []
```

Comment:
Хотел бы обратить внимание на нейминг. Если ты хочешь назвать переменную из 2 слов, тогда разделяй слова `_`, потому что читать переменные без этого становится неудобным и это против PEP8:
```suggestion
num_arr = []
```